app/workers/fast_track/disability_compensation_job.rb

```diff
@@ -64,10 +64,14 @@ def send_fast_track_engineer_email_for_testing(form526_submission_id, error_mess
     end
 
     def get_icn(form526_submission)
-      account = Account.where(idme_uuid: form526_submission.user_uuid).first
-      account = Account.where(logingov_uuid: form526_submission.user_uuid).first if account.blank?
-      account = Account.where(edipi: form526_submission.auth_headers['va_eauth_dodedipnid']).first if account.blank?
-      account.icn if account.present? && account.icn.present?
+      account(form526_submission).icn.presence
```

Comment:
Do we need to call [`#presence`](https://apidock.com/rails/Object/presence) here? I think this is just returning the or `nil` which is the same behavior as `account(form526_submission).icn` right?

Reply:
I used `presence` because we don't want a blank icn - and there's no validation for ICNs on the Account model.

Reply:
It is arguably overly-cautious, I grant you that, since we are handling blank/nil case when initializing the client. 🤷

Reply:
Hm - so I guess the concern here is that it would return something other than nil? If there's no ICN in the account model it would return nil right or does it return an empty string?

Reply:
Totally valid - I'm just making sure I understand.

Reply:
It would return nil with no icn, but the ICN could theoretically be set to a blank string also

 => #<Account id: 1, uuid: "c6e57917-b44f-41c6-bbff-ed24db9e547c", idme_uuid: "6400bbf301eb4e6e95ccea7693eced6f", icn: "", edipi: "1013590059", created_at: "2021-10-26 00:14:24.202283000 +0000", updated_at: "2022-01-06 20:17:41.543335000 +0000", sec_id: "0001740097,0000027647,0000027648", logingov_uuid: nil> 

2.7.4 :008 > a.icn => "" 

2.7.4 :009 > a.valid?
2022-01-06 12:20:56.071340 D [57849:67420 log_subscriber.rb:126] ActiveRecord::Base --   Account Exists? (0.8ms)  SELECT 1 AS one FROM "accounts" WHERE "accounts"."uuid" = $1 AND "accounts"."id" != $2 LIMIT $3  [["uuid", "c6e57917-b44f-41c6-bbff-ed24db9e547c"], ["id", 1], ["LIMIT", 1]]
2022-01-06 12:20:56.075892 D [57849:67420 log_subscriber.rb:126] ActiveRecord::Base --   Account Exists? (1.1ms)  SELECT 1 AS one FROM "accounts" WHERE "accounts"."idme_uuid" = $1 AND "accounts"."id" != $2 LIMIT $3  [["idme_uuid", "6400bbf301eb4e6e95ccea7693eced6f"], ["id", 1], ["LIMIT", 1]]
 => true